spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/actions/TestRemoveOrphanFilesAction.java

```diff
@@ -585,7 +585,7 @@ public void testHiddenPartitionPaths() throws InterruptedException {
     df.write().mode("append").parquet(tableLocation + "/data/_c2_trunc=AA/c3=AAAA");
     df.write().mode("append").parquet(tableLocation + "/data/_c2_trunc=AA/c3=AAAA");
 
-    // Thread.sleep(1000);
+    waitUntilAfter(System.currentTimeMillis() + 1000);
```

Comment:
It should be sufficient to just wait until after `System.currentTimeMillis()`, without the extra delay.